grafana-plugin/src/pages/schedule/Schedule.tsx

```diff
@@ -116,11 +116,13 @@ class SchedulePage extends React.Component<SchedulePageProps, SchedulePageState>
     const users = store.userStore.getSearchResult().results;
     const schedule = scheduleStore.items[scheduleId];
 
-    const disabled =
+    const disabledRotationForm =
       !isUserActionAllowed(UserActions.SchedulesWrite) ||
       schedule?.type !== ScheduleType.API ||
-      shiftIdToShowRotationForm ||
-      shiftIdToShowOverridesForm;
+      shiftIdToShowRotationForm;
```

Comment:
Shouldn't this be converted to boolean instead if we're going to use it such?

```jsx
const disabledRotationForm =
    !isUserActionAllowed(UserActions.SchedulesWrite) ||
    schedule?.type !== ScheduleType.API ||
    !!shiftIdToShowRotationForm;
```

Because `shiftIdToShowRotationForm` doesn't appear to be a bolean, but `disabledRotationForm` is expected to be a boolean at the first glance.